components/Publish.vue

```diff
@@ -160,6 +177,42 @@ export default {
       });
       return false;
     },
+    async wrap() {
+      const signer = await this.provider.getSigner();
+      const signerAddr = await signer.getAddress();
+      if (signerAddr.toLowerCase() != this.ad.owner) {
+        this.error = 'Incorrect active wallet. Must publish with: ' + this.ad.owner;
+        return;
+      }
+      try {
+        const { predictedAddress } = (await this.ketherNFT.precompute(this.ad.idx, signerAddr));
+        // TODO this doesn't acutally work
+        // We need to group these somehow but send one after the first transaction is mined?
```

Comment:
Does it not work? It should queue them both up, each with incrementing nonces. Might have to issue them together and join-await both? (Haven't tried yet)